src/main/java/avd/inf/jdm/rentmycar/domain/Car.java

```diff
@@ -47,14 +47,20 @@ public abstract class Car {
     private String model;
 
     @Column(name="color", nullable = true)
-    private ColorType colorType;
+    private String colorType;
```

Comment:
Voordeel is dat het in de database al als String werd opgeslagen, dus dit heeft buiten de Java Classes geen grote gevolgen! 💪